docs/src/index.md

```diff
@@ -94,6 +94,52 @@ combinations or that you want to break on entry to *any* method of a particular
 At present, note that some of this functionality requires that you be running
 [Revise.jl](https://github.com/timholy/Revise.jl).
 
+It is, in addition, possible to halt execution when otherwise an error would be thrown.
+This functionality is enabled using [`break_on`](@ref) and disabled with [`break_off`](@ref):
+
+```jldoctest demo1
+julia> function f_outer()
+           println("before error")
+           f_inner()
+           println("after error")
+       end;
+
+julia> f_inner() = error("inner error");
+
+julia> JuliaInterpreter.break_on(:error)
```

Comment:
`break_on` and `break_off` are exported. Since we don't qualify `@interpret` etc., let's do the same here.

Reply:
Nice catch, fixed.